kitsune/sumo/static/sumo/scss/form-wizard-sign-in-step.styles.scss

```diff
@@ -1,11 +1,17 @@
+h3 {
+  font-size: 1rem;
+  line-height: 1.5;
```

Comment:
This looks very similar to `h3` styles in the setup device step (I was also using a 26px margin there, but changed it based on how things looked in the copy doc). Maybe we move this to `form-step-styles.scss`?

Reply:
Good idea!